packages/scripts/scripts/check-licenses.js

```diff
@@ -167,14 +167,13 @@ const checkLicense = ( allowedLicense, licenseType ) => {
 	);
 };
 
-// Use `npm ls` to grab a list of all the packages.
+// Use `pnpm ls` to grab a list of all the packages.
```

Comment:
```suggestion
// Use `pnpm list` to grab a list of all the packages.
```

Reply:
Done in https://github.com/WordPress/gutenberg/pull/38624/commits/618ae4ff75845f0d106830dfe0eaa97237ecccbb